### PR TITLE
add ticket button on home page

### PR DESCRIPTION
**Pull request by VictoriaXiao**

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -48,6 +48,54 @@
...
+   <div id="eb-overlay" class=''>
+        <svg id="close-overlay" onclick="overlay_off()" x="0" y="0" viewBox="0 0 24 24" xml:space="preserve">
+            <path id="cross-chunky_svg__eds-icon--cross-chunky_base"
+                d="M13.4 12l3.5-3.5-1.4-1.4-3.5 3.5-3.5-3.5-1.4 1.4 3.5 3.5-3.5 3.5 1.4 1.4 3.5-3.5 3.5 3.5 1.4-1.4z">
+            </path>
+        </svg>
+        <div id="eventbrite-widget-container" style="width:100%">
+        </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```
[the remark this answers is not in the document]
Comment (zzhai) [reply _layouts/home.html:56]: What is the code block below?

[the remark this answers is not in the document]
Comment (zzhai) [reply _layouts/home.html:56]: _<path id="cross-chunky_svg__eds-icon--cross-chunky_base"
                d="M13.4 12l3.5-3.5-1.4-1.4-3.5 3.5-3.5-3.5-1.4 1.4 3.5 3.5-3.5 3.5 1.4 1.4 3.5-3.5 3.5 3.5 1.4-1.4z">
            </path>_

[the remark this answers is not in the document]
Comment (VictoriaXiao) [reply _layouts/home.html:56]: > ```
>         _<path id="cross-chunky_svg__eds-icon--cross-chunky_base"
>             d="M13.4 12l3.5-3.5-1.4-1.4-3.5 3.5-3.5-3.5-1.4 1.4 3.5 3.5-3.5 3.5 1.4 1.4 3.5-3.5 3.5 3.5 1.4-1.4z">
>         </path>_  
> ```

Looks like this is about the eventbrite widget.

[the remark this answers is not in the document]
Comment (zzhai) [reply _layouts/home.html:56]: where is the reply?

[the remark this answers is not in the document]
Comment (VictoriaXiao) [reply _layouts/home.html:56]: Looks like this is about the eventbrite widget.

[the remark this answers is not in the document]
Comment (zzhai) [reply _layouts/home.html:56]: It doesn't look like belonging to Eventbrite widget. 
Seems to be extra formatting added by Github.

[the remark this answers is not in the document]
Comment (VictoriaXiao) [reply _layouts/home.html:56]: > 

Updated it.

**Review by zzhai — APPROVED** [review]

**Pull request merged**